FMB/FiniteModelBuilder.cpp

```diff
@@ -1673,7 +1673,7 @@ MainLoopResult FiniteModelBuilder::runImpl()
 #endif
     //TODO consider adding clauses directly to SAT solver in new interface?
     // pass clauses and assumption to SAT Solver
-    SATSolver::Status satResult;
```

Comment:
Apparently `satResult` could be used unitialised. I don't think it can, but this doesn't hurt.